chromium_src/chrome/browser/resources/app_settings/window_mode_item.ts

```diff
@@ -0,0 +1,83 @@
+// Copyright (c) 2025 The Brave Authors. All rights reserved.
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this file,
+// You can obtain one at https://mozilla.org/MPL/2.0/.
+
+import type { App } from '//resources/cr_components/app_management/app_management.mojom-webui.js';
+import { CrLitElement, css } from '//resources/lit/v3_0/lit.rollup.js';
+
+import { getCss } from './app_management_shared_style.css.js';
+import { createDummyApp } from './web_app_settings_utils.js';
+import { getHtml } from './window_mode_item.html.js';
+import { loadTimeData } from '//resources/js/load_time_data.js';
+import { BrowserProxy } from '//resources/cr_components/app_management/browser_proxy.js';
+import { WindowMode } from '//resources/cr_components/app_management/constants.js';
+
+export class WindowModeItemElement extends CrLitElement {
+  static get is() {
+    return 'app-management-window-mode-item';
+  }
+
+  static override get styles() {
+    return [
+      getCss(),
+      css`
+        :host > div {
+          width: 100%;
+        }
+
+        leo-radiobutton {
+          display: flex;
+        }
+      `
+    ];
+  }
+
+  override render() {
+    return getHtml.bind(this)();
+  }
+
+  static override get properties() {
+    return {
```

Comment:
I was about to comment about how perhaps we shouldn't replace the class but subclass and modify but it does seem like we're removing most of the upstream functionality so I think it crosses the line between replace and override 🤔

Reply:
I think the fact we replace the whole HTML for this element makes me more comfortable with this approach